Tests/SentryTests/Networking/SentryHttpTransportFlushIntegrationTests.swift

```diff
@@ -96,24 +102,19 @@ final class SentryHttpTransportFlushIntegrationTests: XCTestCase {
         waitForEnvelopeToBeStored(dispatchQueueWrapper)
         requestManager.returnResponse(response: HTTPURLResponse())
 
-        let flushTimeout = 0.1
-        requestManager.waitForResponseDispatchGroup = true
-        requestManager.responseDispatchGroup.enter()
-
-        let allFlushCallsGroup = DispatchGroup()
+        let initialFlushCallGroup = DispatchGroup()
```

Comment:
`m`: I currently remove all DispatchGroups from the tests and replace them with XCTestExpectations cause these fail when they time out and we can also configure them to fail when they get called too often. This is part of https://github.com/getsentry/sentry-cocoa/issues/5789. But we can also follow up on this.